src/main/java/com/google/devtools/build/lib/rules/java/JavaConfiguration.java

```diff
@@ -211,6 +212,7 @@ public boolean alwaysGenerateOutputMapping() {
     this.isJlplStrictDepsEnforced = javaOptions.isJlplStrictDepsEnforced;
     this.disallowResourceJars = javaOptions.disallowResourceJars;
     this.addTestSupportToCompileTimeDeps = javaOptions.addTestSupportToCompileTimeDeps;
+    this.useRemoteJavaToolchain = javaOptions.useRemoteJavaToolchain;
```

Comment:
doesn't this to happen before line 199?

Reply:
Yes sorry, that method was leftover - i moved all the logic in JavaOptions. JavaConfiguration doesn't know anything about these flags anymore.